src/text/2-package.tex

```diff
@@ -1,6 +1,105 @@
 \section{Analysis Workflow Overview}
 \label{sec:analysis-workflow-overview}
-\todo{Regis and Fabio F.}
+
+% This might not be the bast place compared to introduction
+The data processing workflow in Gamma-ray astronomy is usually split in two sequences.
+The first one deals with the data processing from camera measurement, calibration, event
+reconstruction and selection to yield a list of reconstructed gamma-ray event candidates.
+This sequence, sometimes referred to as low-level analysis, is usually very specific to
+a given observation technique if not a given instrument.
+
+The other sequence, referred to as high-level analysis, deals with the extraction of physical
+quantities related to gamma-ray sources and the production of high-level products such as spectra,
+lightcurves and catalogs. The methods and tools applied here are more generic and are broadly
+shared across the field. They also frequently imply joint analysis of multi-instrument data.
+To extract physically relevant information, the measured data are usually compared to a
```

Comment:
==> To extract the physically relevant information of the observed gamma-ray sources, the measured data are compared to a sky model using statistical techniques such as maximum likelihood.